### PR TITLE
docs: Changes to settings docs for v1

### PR DESCRIPTION
**Pull request by chrisnegus**
**Description**: Updated descriptions in docs for [Settings](https://karpenter.sh/preview/reference/settings/) that changed for v1.

**Does this change impact docs?**
- [X ] Yes, PR includes docs updates 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/reference/settings.md [hunk 1]
```diff
@@ -28,15 +26,15 @@ Karpenter surfaces environment variables and CLI parameters to allow you to conf
 | KARPENTER_SERVICE | \-\-karpenter-service | The Karpenter Service name for the dynamic webhook certificate|
 | KUBE_CLIENT_BURST | \-\-kube-client-burst | The maximum allowed burst of queries to the kube-apiserver (default = 300)|
 | KUBE_CLIENT_QPS | \-\-kube-client-qps | The smoothed rate of qps to kube-apiserver (default = 200)|
-| LEADER_ELECT | \-\-leader-elect | Start leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.|
+| DISABLE_LEADER_ELECTION | \-\-disable-leader-election | Start leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.|
```
Comment (rschalo) [inline website/content/en/preview/reference/settings.md:29]:
```suggestion
| DISABLE_LEADER_ELECTION | \-\-disable-leader-election | Disable the leader election client. Disable this when running a single replica components. (default = false)|
```

Comment (jonathan-innis) [reply website/content/en/preview/reference/settings.md:29]: Did we run `make docgen` on this? It should automatically remove these and update their descriptions IIRC

File: website/content/en/preview/reference/settings.md [hunk 2]
```diff
@@ -28,15 +26,15 @@ Karpenter surfaces environment variables and CLI parameters to allow you to conf
...
-| METRICS_PORT | \-\-metrics-port | The port the metric endpoint binds to for operating metrics about the controller itself (default = 8000)|
+| METRICS_PORT | \-\-metrics-port | The port the metric endpoint binds to for operating metrics about the controller itself (default = 8080)|
 | RESERVED_ENIS | \-\-reserved-enis | Reserved ENIs are not included in the calculations for max-pods or kube-reserved. This is most often used in the VPC CNI custom networking setup https://docs.aws.amazon.com/eks/latest/userguide/cni-custom-network.html. (default = 0)|
 | VM_MEMORY_OVERHEAD_PERCENT | \-\-vm-memory-overhead-percent | The VM memory overhead as a percent that will be subtracted from the total memory for all instance types. (default = 0.075)|
-| WEBHOOK_METRICS_PORT | \-\-webhook-metrics-port | The port the webhook metric endpoing binds to for operating metrics about the webhook (default = 8001)|
+| WEBHOOK_METRICS_PORT | \-\-webhook-metrics-port | The port the webhook metric endpoing binds to for operating metrics about the webhook (default = 8081)|
```
Comment (rschalo) [inline website/content/en/preview/reference/settings.md:37]: We ended up not making this change. I think there's a few more things we'll be doing but then we'll run `make docgen` again.

**Comment by jonathan-innis** [comment]
Closing since this was handled by an earlier PR

**Pull request closed without merging**